include/klee/Internal/ADT/TreeStream.h

```diff
@@ -18,6 +18,19 @@ namespace klee {
   typedef unsigned TreeStreamID;
   class TreeOStream;
 
+  struct PathLocation {
```

Comment:
@MartinNowack It's a good point, and some time ago I implemented this feature using KInstruction. The main issue (but maybe is due to my limited development skills) is the replay feature: do you want to instantiate (reading from the file!) the KInstruction? I don't think it is a good idea, but maybe you already have a better option.